src/zrc/datamodel/constants.py

```diff
@@ -81,33 +81,40 @@ class TyperingInrichtingselement(DjangoChoices):
 
 
 class TyperingKunstwerk(DjangoChoices):
-    keermuur = ('Keermuur', 'Keermuur')
-    overkluizing = ('Overkluizing', 'Overkluizing')
-    duiker = ('Duiker', 'Duiker')
-    faunavoorziening = ('Faunavoorziening', 'Faunavoorziening')
-    vispassage = ('Vispassage', 'Vispassage')
-    bodemval = ('Bodemval', 'Bodemval')
-    coupure = ('Coupure', 'Coupure')
-    ponton = ('Ponton', 'Ponton')
-    voorde = ('Voorde', 'Voorde')
-    hoogspanningsmast = ('Hoogspanningsmast', 'Hoogspanningsmast')
-    gemaal = ('Gemaal', 'Gemaal')
-    perron = ('Perron', 'Perron')
-    sluis = ('Sluis', 'Sluis')
-    strekdam = ('Strekdam', 'Strekdam')
-    steiger = ('Steiger', 'Steiger')
-    stuw = ('Stuw', 'Stuw')
+    keermuur = ChoiceItem('Keermuur', 'Keermuur')
+    overkluizing = ChoiceItem('Overkluizing', 'Overkluizing')
+    duiker = ChoiceItem('Duiker', 'Duiker')
+    faunavoorziening = ChoiceItem('Faunavoorziening', 'Faunavoorziening')
+    vispassage = ChoiceItem('Vispassage', 'Vispassage')
+    bodemval = ChoiceItem('Bodemval', 'Bodemval')
+    coupure = ChoiceItem('Coupure', 'Coupure')
+    ponton = ChoiceItem('Ponton', 'Ponton')
+    voorde = ChoiceItem('Voorde', 'Voorde')
+    hoogspanningsmast = ChoiceItem('Hoogspanningsmast', 'Hoogspanningsmast')
+    gemaal = ChoiceItem('Gemaal', 'Gemaal')
+    perron = ChoiceItem('Perron', 'Perron')
+    sluis = ChoiceItem('Sluis', 'Sluis')
+    strekdam = ChoiceItem('Strekdam', 'Strekdam')
+    steiger = ChoiceItem('Steiger', 'Steiger')
+    stuw = ChoiceItem('Stuw', 'Stuw')
 
 
 class TyperingWater(DjangoChoices):
-    zee = ('Zee', 'Zee')
-    waterloop = ('Waterloop', 'Waterloop')
-    watervlakte = ('Watervlakte', 'Watervlakte')
-    greppel_droge_sloot = ('Greppel, droge sloot', 'Greppel, droge sloot')
+    zee = ChoiceItem('Zee', 'Zee')
+    waterloop = ChoiceItem('Waterloop', 'Waterloop')
+    watervlakte = ChoiceItem('Watervlakte', 'Watervlakte')
+    greppel_droge_sloot = ChoiceItem('Greppel, droge sloot', 'Greppel, droge sloot')
 
 
 class TypeSpoorbaan(DjangoChoices):
     breedspoor = ChoiceItem('breedspoor')
     normaalspoor = ChoiceItem('normaalspoor')
     smalspoor = ChoiceItem('smalspoor')
     spoorbaan = ChoiceItem('spoorbaan')
+
+
+class IndicatieMachtiging(DjangoChoices):
+    gemachtigde = ChoiceItem('gemachtigde', _("De betrokkene in de rol bij de zaak is door een andere betrokkene bij "
```

Comment:
Can we get these descriptions in the help_text? Like `betalingsindicatie` in  https://ref.tst.vng.cloud/zrc/api/v1/schema/#operation/zaak_create

Reply:
description of `IndicatieMachtiging` choices is added to the serializer